package.json

```diff
@@ -49,7 +49,8 @@
     "docs": "babel-node ./scripts/documentation.js",
     "example-version": "babel-node ./scripts/edit-version.js",
     "prettier-all": "prettier --write '{src,examples,test}/**/*.js'",
-    "lint": "eslint src test webpack examples/*.js examples/**/src website/*.js website/src --fix",
+    "lint": "yarn typescript && eslint src test webpack examples/*.js examples/**/src website/*.js website/src --fix",
+    "typescript": "tsc",
```

Comment:
To be clear we can write `tsc --noEmit` - though we do have the noEmit setting in the tsconfig, I still think it is worth having it here to emphasize that we are just type checking.